### DISCUSSION TITLE
Kibana: Server not ready

### DISCUSSION THREAD

**Discussion by Maintech3** [Security-Onion-Solutions/securityonion#5073] [Unsupported Versions]
After last update kibana will not work for me. Using the cgi interface I get an error 404 when attempting to start kibana. I then went to the cli interface and entered "sudo so-kibana-config-load. The output read "Server is not ready. Waiting for value 'Elastic' at 'http://localhost:5xxx/app/kibana'" I then went to the log files mentioned in another post to see if I could find where it was looking for Kibana. There was nothing. I went to the config file for kibana and the word kibana was used many times and I could not determine if there was anything missing. I then followed your online doc and tried "curl -k -XDELETE https://localhost:9200/.kibana_6. The output was "Curl: (51) Empty reply from server.
I have no idea where to look now. Also I am no programmer. Any ideas? If there are any online instructions please just post the url.
I appreciate any help and thank you in advance.

**Comment by Git-Me-Some-Hub** [comment]
This is a little old, but...
- I will assume you restarted the machine at this time. 
- Did you do so-status to confirm that Kibana is running? 
- Did you look at the log file? (/opt/so/log/kibana/)
- Per the Forum rules you need to say what type of install and the version you are dealing with.
- - What version were you at and what version did you go to. 
- - Also very helpful to understand the setup, single or distributed. if the later, how many nodes, and what types (heavy, manger, search,...) 
Just thought this might help someone else in the future.
